apps/nextjs/src/app/api/aila-download-all/route.ts

```diff
@@ -132,6 +132,7 @@ async function getHandler(req: Request): Promise<Response> {
           lessonExport.exportType,
         )}.${ext}`;
 
+        // @ts-expect-error @todo fix this
```

Comment:
@tomwisecodes I think you wrote this part? It's passing in a ReadableStream which isn't supported by the library and only became apparent when I added the types for the archive package

Reply:
@JBR90 this looks relevant to your PR